src/components/LandingPage/LandingPage.module.scss

```diff
@@ -46,18 +51,26 @@
 }
 
 //tablet
-@media (min-width: 480px) and (max-width: 768px) {
+@media (min-width: 480px) and (orientation: portrait) { //jag blev tvungen att lägga till orientation för bakgrundsbilder
```

Comment:
This is not a bad thing :) However, turn comment into a generic format, and rather add these notes in the PR (you could comment on your own PR).

I.e. comment in code:
"Orientation for background images"

And then in the PR your current code comment :)

Reply:
Tack! Hade inte koll på att man kan kommentera på kod på github.